receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -150,8 +155,107 @@ func (prw *prometheusRemoteWriteReceiver) parseProto(contentType string) (promco
 }
 
 // translateV2 translates a v2 remote-write request into OTLP metrics.
-// For now translateV2 is not implemented and returns an empty metrics.
+// translate is not feature complete.
 // nolint
-func (prw *prometheusRemoteWriteReceiver) translateV2(_ context.Context, _ *writev2.Request) (pmetric.Metrics, promremote.WriteResponseStats, error) {
-	return pmetric.NewMetrics(), promremote.WriteResponseStats{}, nil
+func (prw *prometheusRemoteWriteReceiver) translateV2(_ context.Context, req *writev2.Request) (pmetric.Metrics, promremote.WriteResponseStats, error) {
+	var (
+		badRequestErrors error
+		otelMetrics      = pmetric.NewMetrics()
+		labelsBuilder    = labels.NewScratchBuilder(0)
+		stats            = promremote.WriteResponseStats{}
+	)
+
+	for _, ts := range req.Timeseries {
+		ls := ts.ToLabels(&labelsBuilder, req.Symbols)
+
+		if !ls.Has(labels.MetricName) {
+			badRequestErrors = errors.Join(badRequestErrors, fmt.Errorf("missing metric name in labels"))
+			continue
+		} else if duplicateLabel, hasDuplicate := ls.HasDuplicateLabelNames(); hasDuplicate {
+			badRequestErrors = errors.Join(badRequestErrors, fmt.Errorf("duplicate label %q in labels", duplicateLabel))
+			continue
+		}
+
+		var rm pmetric.ResourceMetrics
+		// This cache should be populated by the metric 'target_info', but we're not handling it yet.
+		hashedJobAndInstance := xxhash.Sum64String(ls.Get("job") + string([]byte{'\xff'}) + ls.Get("instance"))
+		cacheEntry, ok := prw.jobInstanceCache[hashedJobAndInstance]
+		if ok {
+			rm = pmetric.NewResourceMetrics()
```

Comment:
> I'm experimenting more and more in my local branch, and it feels like caching between requests is just not the right move at this moment. I'll update the pull request to cache the time series in the same request. Does that sound good?

That sounds good to me.  We will need to add it back in eventually, but since we can't test it without target_info i'm definitely OK to skip it for now.